manifests/logging-stack/auditd/output-auditd-elasticsearch.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   matchRegex: auditd
   es:
-    host: elasticsearch-logging-data.kubesphere-logging-system.svc
+    host: elasticsearch-logging-data.fluent.svc
```

Comment:
we can keep the es namespace as it is before : elasticsearch-logging-data.kubesphere-logging-system.svc